walkthroughs/week-1-setup/intro-walkthrough.md

```diff
@@ -125,7 +125,6 @@ get more specific instructions each week.
 You finished this introduction!
 
 Next, you'll write HTML and JavaScript to create a portfolio page, and you'll
-deploy it to App Engine so anyone can visit it.
```

Comment:
is deleting this line an accident? :)

Reply:
![Screenshot 2020-05-29 at 5 30 50 PM - Display 1](https://user-images.githubusercontent.com/39624330/83307258-52a57280-a1d2-11ea-9e2c-c2519e7be9ab.png)
![Screenshot 2020-05-29 at 5 31 14 PM - Display 1](https://user-images.githubusercontent.com/39624330/83307269-5933ea00-a1d2-11ea-90d5-ee2feeb57ec8.png)
yeah, sorry is there an easy way to undo it?

Reply:
I think you'll just have to type out the line again :)